src/components/SignInForm.jsx

```diff
@@ -24,30 +24,29 @@ const SignIn = (props, routeProps) => {
 	if (IsFormInComplete(props) && !ignoreFormCompletion) {
 		GoHome(props);
 	}
-	
-	const hasAlertMessage = () =>{
+
+	const hasAlertMessage = () => {
```

Comment:
this should probably be called `getAlertMessage` . hasAlertMessage sounds like it will return a boolean.